src/script/auth/module/action/AuthAction.ts

```diff
@@ -385,7 +385,10 @@ export class AuthAction {
     return async (dispatch, getState, {core}) => {
       dispatch(AuthActionCreator.startValidateLocalClient());
       try {
-        await core.loadAndValidateLocalClient();
+        const {
+          authState: {entropy},
+        } = getState();
```

Comment:
i dont think you should assume the entropy has landed here, in several cases it is not added to the store currently. better to pass it into doInit/ or verify it has been added to the redux store in the Login and conversationJoin cases